### ISSUE TITLE
Enhancement: Adjust Stats ALT Attribute to Reduce Errors in Validation Tools

### ISSUE THREAD

**Issue by bobmatyas** [Automattic/jetpack#23777]
### Impacted plugin

Jetpack

### What

Instead of rendering an emtpty `alt` attribute in the tracking stats, render `alt=""` instead

### How

While the current Jetpack Stats code is technically correct (validates and doesn't trigger errors in WAVE WebAim):

<img width="821" alt="Screen Shot on 2022-03-23 at 15-56-57" src="https://user-images.githubusercontent.com/45246438/161601457-63b0fa4b-4365-429c-abc2-0e84040626ee.png">

in some automated tools, the empty attribute `alt` causes errors. It has been requested that we replace this with `alt=""`

Related case: https://github.com/Automattic/jetpack/issues/2856

**Comment by bobmatyas** [comment]
Reported in 4920822-zen

**Comment by jeherve** [comment]
I'm afraid we may not have power over this. We do define an empty `alt` attribute. It is then up to the browser to build the image HTML tag. In this case, the browser, when given an empty string for an attribute, adds that attribute as an empty one.

Is the problem happening with a specific tool? It may be worth reaching out to them and asking if they could detect that empty attribute?

**Comment by jeherve** [comment]
Closing this for now because of the lack of activity on this. We can always reopen in the future if needed.
